### PR TITLE
Eng 1316 Update comments for all server handlers to follow handler-comment templates

### PR DESCRIPTION
**Pull request by likawind**
## Describe your changes and why you are making these changes
This PR adds comments to handlers to follow our handler comment templates.

## Related issue number (if any)
ENG-1316

## Checklist before requesting a review
Shouldn't need much tests as there's no feature change.
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/golang/cmd/server/handler/list_workflow_objects.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 //		`api-key`: user's API Key
 // Response:
 //	Body:
```
Comment (eunice-chan) [inline src/golang/cmd/server/handler/list_workflow_objects.go:25]: What's the difference between Response: [something] vs Response: Body: [something]?

**Review by eunice-chan — APPROVED** [review]
LGTM! Thanks for doing this!

File: src/golang/cmd/server/handler/list_workflow_objects.go (hunk 1 above)
Comment (likawind) [reply src/golang/cmd/server/handler/list_workflow_objects.go:25]: They are the same :) Updated some comments so that all are consistent (have `Body`)

Comment (likawind) [reply src/golang/cmd/server/handler/list_workflow_objects.go:25]: Encountered some linting issues on spacing, I'm merging with the old version without add `Body`s. Could come back to this after we got linting issues resolved

**Pull request merged**